CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+### [1.0.2] - 2022-01-17
```

Comment:
See line 6?

Reply:
As there is a bug in JWT addon, so will be releasing it as well.

Reply:
Shall i remove line 6?

Reply:
I guess that’s your call. I’m the zap repos the version and date are inserted as part of the release process (replacing unreleased) then the unreleased heading is re-added and manifest (or build file) version incremented in a subsequent step/PR.